src/dev/build/tasks/os_packages/docker_generator/templates/build_docker_sh.template.ts

```diff
@@ -54,10 +56,12 @@ function generator({
   retry_docker_pull ${baseOSImage}
 
   echo "Building: kibana${imageFlavor}-docker"; \\
-  docker build -t ${imageTag}${imageFlavor}:${version} -f Dockerfile . || exit 1;
+  docker build -t ${imageTag}${imageFlavor}:${version}${dockerTagQualifier} -f Dockerfile . || exit 1;
```

Comment:
It would make it a bit easier to follow this if we assigned the dockerName to a variable to re-use.

Reply:
Pushed with 9634ed3e7d467387d6e43e04293a1c741ba7a97d